src/recipes/common.recipe.ts

```diff
@@ -0,0 +1,49 @@
+import { ArrayMinSize, ArrayMaxSize, IsArray, Validate } from 'class-validator';
+
+import { ArraySizeOptions, DecoratorOptions, ValidateOptions } from '../options';
+
+export function buildArrayPropertyDecorators(
```

Comment:
This logic was defined in another PR (#32). This file now contains multiple bits of common/shared validation logic.